Sources/_StringProcessing/Compiler.swift

```diff
@@ -101,6 +109,60 @@ class Compiler {
     }
   }
 
+  func compileLookaround(
```

Comment:
```suggestion
  func emitLookaround(
```
Nit: How about starting with "emit" as existing subroutines called from `emit(_:)` are named "emit...".